src/components/license-activation/LicenseActivation.jsx

```diff
@@ -50,7 +50,6 @@ export default function LicenseActivation() {
     <>
       <Helmet title={PAGE_TITLE} />
       <div className="container-fluid py-5">
-        <p>{LOADING_MESSAGE}</p>
```

Comment:
note: removing the rendered text since this message is only displayed for the duration of the API request, which gives very little time for the user to actually read the message.

Reply:
very fair